### DISCUSSION TITLE
Tinted glass

### DISCUSSION THREAD

**Discussion by karimknaebel** [mitsuba-renderer/mitsuba3#568] [Q&A] [answered]
Are `principled` and `principledthin` the correct BSDFs to render "tinted" glass?

**Comment by njroussel** [comment, marked as answer]
I'm not sure what the "correct" BSDF would be. You can indeed achieve that look with those BSDFs. Also, you might want to use the `specular_reflectance/transmittance` parameters in the `dielectric` and `thindielectric` plugins. 
FWIW, we also implement a tinted dielectric in [this tutorial](https://mitsuba.readthedocs.io/en/stable/src/others/custom_plugin.html).

**Reply by karimknaebel** [reply]
thanks! :)
